docs/apache-airflow-providers-elasticsearch/logging/index.rst

```diff
@@ -49,7 +49,7 @@ To output task logs to stdout in JSON format, the following config could be used
 
 .. _write-logs-elasticsearch-tls:
 
-Writing logs to Elasticsearch over TLS
+Reading logs from Elasticsearch over TLS
 ''''''''''''''''''''''''''''''''''''''
```

Comment:
Need to fix docs formats. I would recommend you check it locally in [Breeze](https://github.com/apache/airflow/blob/main/BREEZE.rst#building-the-documentation) before push changes, it takes less time for re-run CI, because it is your first contribution you need some one who has write access to repo to manually approve CI run for you 

 
```console
WARNING: Title underline too short.

File path: apache-airflow-providers-elasticsearch/logging/index.rst (53)

  48 |     json_format = True
  49 | 
  50 | .. _write-logs-elasticsearch-tls:
  51 | 
  52 | Reading logs from Elasticsearch over TLS
> 53 | ''''''''''''''''''''''''''''''''''''''
  54 | 
  55 | To add custom configurations to ElasticSearch (e.g. turning on ``ssl_verify``, adding a custom self-signed
  56 | cert, etc.) use the ``elasticsearch_configs`` setting in your ``airflow.cfg``
  57 | 
  58 | .. code-block:: ini
```